EKF/control.cpp

```diff
@@ -1331,7 +1328,7 @@ void Ekf::controlFakePosFusion()
 		_using_synthetic_position = true;
 
 		// Fuse synthetic position observations every 200msec
-		if (isTimedOut(_time_last_fake_pos, (uint64_t)2e5) || _fuse_height) {
+		if (isTimedOut(_time_last_fake_pos, (uint64_t)2e5)) {
```

Comment:
My interpretation of the intent here was that we wanted to fuse the horizontal fake position the same amount as we fuse the baro data which we assume is available. Not sure if this is necessary.

Reply:
That's what I also thought, but then I think we have to do it differently because with that, we're fusing every 200ms + every baro update. We can discuss about that in the next call but for now, it seems fine not to do it.

Reply:
And the temptation to remove a global variable was too big....

Reply:
Yes, I completely agree. If you feel comfortable to do so, please go ahead.